src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends FigureColor {
```

Comment:
The class 'Circle' should not extend 'FigureColor'; it seems like 'FigureColor' is being used to set behavior for classes, which is not recommended. Consider using an interface to define behavior and a separate class to handle color properties.